### PR TITLE
stream aggregation: support dropping multiple labels per `remoteWrite…

### PR DESCRIPTION
**Pull request by Haleygo**
….url` using `-remoteWrite.streamAggr.dropInputLabels`

Before, labels were set to all the `remoteWrite.url`.

address https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6780

### CODE REVIEW

File: docs/changelog/CHANGELOG.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ See also [LTS releases](https://docs.victoriametrics.com/lts-releases/).
 * BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert/): properly update value of variable `$activeAt` in rules annotation during replay mode. Before, `$activeAt` could have provided incorrect values during replay.
 * BUGFIX: [MetricsQL](https://docs.victoriametrics.com/metricsql/): properly handle `c1 AND c2` and `c1 OR c1` queries for constants `c1` and `c2`. Previously such queries could return unexpected results. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6637).
 * BUGFIX: all VictoriaMetrics components: increase default value of `-loggerMaxArgLen` cmd-line flag from 1000 to 5000. This should improve visibility on errors produced by very long queries.
+* BUGFIX: [stream aggregation](https://docs.victoriametrics.com/stream-aggregation/): support having multiple labels per `remoteWrite.url` using `-remoteWrite.streamAggr.dropInputLabels`. Before, labels were set to all the `remoteWrite.url`. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6780) for the details.
```
Comment (AndrewChubatiuk) [inline docs/changelog/CHANGELOG.md:46]: @Haleygo should we also mention somewhere that this change breaks previous logic?

Comment (Haleygo) [reply docs/changelog/CHANGELOG.md:46]: In https://github.com/VictoriaMetrics/VictoriaMetrics/commit/57183c9b612a06685be2bc07d879743c75d2be82,  the bugfix description was removed from `update note`. 
This pull request should probably follow the same rule, or both of them should be mentioned in `update note`. I'd like to confirm with @hagen1778

**Review by f41gh7 — APPROVED** [review]
LGTM

File: docs/changelog/CHANGELOG.md [hunk 2]
```diff
@@ -32,6 +32,7 @@ See also [LTS releases](https://docs.victoriametrics.com/lts-releases/).
...
 * BUGFIX: [vmagent](https://docs.victoriametrics.com/vmagent/): properly consume messages [from kafka](https://docs.victoriametrics.com/vmagent/#kafka-integration). Previously vmagent could skip some messages during start-up.
 * BUGFIX: [stream aggregation](https://docs.victoriametrics.com/stream-aggregation/): perform deduplication for all received data when specifying `-streamAggr.dedupInterval` or `-remoteWrite.streamAggr.dedupInterval` command-line flags are set. Previously, if the `-remoteWrite.streamAggr.config` or `-streamAggr.config` is set, only series that matched aggregation config were deduplicated. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6711#issuecomment-2288361213) for details.
+* BUGFIX: [stream aggregation](https://docs.victoriametrics.com/stream-aggregation/): support having multiple labels per `remoteWrite.url` using `-remoteWrite.streamAggr.dropInputLabels`. Before, labels were set to all the `remoteWrite.url`. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6780) for the details.
```
Comment (hagen1778) [inline docs/changelog/CHANGELOG.md:35]:
```suggestion
* BUGFIX: [stream aggregation](https://docs.victoriametrics.com/stream-aggregation/): fix  `-remoteWrite.streamAggr.dropInputLabels` labels parsing. Now, this flag allows specifying a list of labels to drop (by using '^^' separator, i.e. `dropInputLabels='replica^^az,replica'`) per each corresponding `remoteWrite.url`. Before, `-remoteWrite.streamAggr.dropInputLabels` labels were incorrectly applied to all configured `remoteWrite.url`s. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6780) for the details.
```

**Review by hagen1778 — APPROVED** [review]
Thanks!

**Pull request merged**